### ISSUE TITLE
[Package Request]: dploy

### ISSUE THREAD

**Issue by soredake** [microsoft/winget-pkgs#37630]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [ ] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: arecarn
Publisher Url: https://github.com/arecarn/dploy
Package Name: dploy
Description:  Provides the functionality of GNU Stow in a cross platform CLI tool and python module 
Package Version: 0.1.2
Installer URL: https://files.pythonhosted.org/packages/d7/7c/36cfbb3bf2db3e5d56a4d176b162ce52fe78b5a8fbeb9d19daaadc3fafac/dploy-0.1.2.tar.gz

Current Package Identifier (for new versions):

**Comment by denelon** [comment]
The Windows Package Manager currently requires either an MSIX, MSI, or an .exe based installer. Support for zipped/compressed files is not supported.

**Comment by lychichem** [comment]
> The Windows Package Manager currently requires either an MSIX, MSI, or an .exe based installer. Support for zipped/compressed files is not supported.

So we need to change the issue template to "not a compressed file", or users will submit issues like this, and say: "hey guys, I found a great software, and it's not a .zip file." At least, the issue template now can only make me think like this. What's your opinion, @denelon ?

**Comment by OfficialEsco** [comment]
Like this? 😁
![image](https://user-images.githubusercontent.com/15158490/146656902-39a36d55-f462-4280-80cb-9f5b118ef423.png)

**Comment by lychichem** [comment]
> Like this? 😁
> ![image](https://user-images.githubusercontent.com/15158490/146656902-39a36d55-f462-4280-80cb-9f5b118ef423.png)
> 

.tar, .tar.bz2, .gz, .bin, or any other compressed files should be listed alongside zip. We need to list all the forbidden types of files, or we should tell users we just allow few types. I mean, the description in checkbox should change.

**Comment by denelon** [comment]
I'm happy to look at a PR for changing the template. I'm good with what you think would work best.

**Comment by Trenly** [comment]
Even though .zip is unblocked, .tar.gz was not part of the initial implementation. This is still blocked by https://github.com/microsoft/winget-cli/issues/2899
